javascript-loops-practice/javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,12 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  const result = [];
+
+  for (const student of students) {
+    for (const name in student) {
+      result.push(student[name]);
+    }
+  }
+
+  return result;
+}
```

Comment:
This works, but we want to be very careful when using multiple loops. We want to try and avoid nested loops as much as possible. The approach below is how I would've handled this.
```
function getStudentNames(students) {
  const result = [];
  for (let i = 0; i < students.length; i++) {
    result.push(students[i].name);
  }
  return result;
}
```

Reply:
got it, thanks for the feedback!